clickup/user_groups.go

```diff
@@ -0,0 +1,58 @@
+package clickup
+
+import (
+	"context"
+)
+
+type UserGroupsService service
+
+type GetUserGroupsResponse struct {
+	UserGroups []UserGroup `json:"groups"`
+}
+
+type UserGroup struct {
+	ID          string        `json:"id"`
+	TeamID      string        `json:"team_id"`
+	UserID      int           `json:"userid"`
+	Name        string        `json:"name"`
+	Handle      string        `json:"handle"`
+	DateCreated string        `json:"date_created"`
+	Initials    string        `json:"initials"`
+	Members     []GroupMember `json:"members"`
+	Avatar      interface{}   `json:"avatar"`
+}
+
+type GroupMember struct {
+	ID             int    `json:"id"`
+	Username       string `json:"username"`
+	Email          string `json:"email"`
+	Color          string `json:"color"`
+	Initials       string `json:"initials"`
+	ProfilePicture string `json:"profilePicture"`
+}
+
+type GetUserGroupsOptions struct {
+	TeamID  string `url:"team_id,omitempty"`
+	GroupID string `url:"group_id,omitempty"`
+}
```

Comment:
Is this parameter (`group_id`) correct? According to the documentation, `group_ids` may be correct.

> Enter one or more Team ids (user groups) to retrieve information about specific Teams.
https://clickup.com/api/clickupreference/operation/GetTeams1/

```suggestion
type GetUserGroupsOptions struct {
	TeamID  string `url:"team_id,omitempty"`
	GroupIDs string `url:"group_ids,omitempty"`
}
```

Reply:
Hey! Yes, you are correct, I missed it. Fixed it.